cron-jobs/packit-service-validation/packit-service-validation.py

```diff
@@ -378,21 +441,31 @@ def check_comment(self):
     else:
         logging.warning("SENTRY_SECRET was not set!")
 
+    if getenv("DEPLOYMENT"):
+        deployment = (
+            Staging() if getenv("DEPLOYMENT") == Deployments.staging else Production()
+        )
+    else:
+        deployment = Staging()
+        # deployment = Production()
+
```

Comment:
```suggestion
deployment = Production() 
                      if getenv("DEPLOYMENT", Deployments.production) == Deployments.production
                      else Staging()
```
I am also thinking about whether the default should be staging or production :thinking: but the `Testcase` class uses production as default

Reply:
I also prefer set `Production` as default, you were such speedy in these review that I had not time to fix it from my test version ^_^